pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -218,8 +218,7 @@ let
       ++ lib.optionals pythonSupport [ "--with-python" ]
       ++ lib.optionals jitSupport [ "--with-llvm" ]
       ++ lib.optionals pamSupport [ "--with-pam" ]
-      # This could be removed once the upstream issue is resolved:
-      # https://postgr.es/m/flat/427c7c25-e8e1-4fc5-a1fb-01ceff185e5b%40technowledgy.de
+      # This can be removed once v16 is removed. v17+ ships with it.
       ++ lib.optionals (stdenv'.hostPlatform.isDarwin && atLeast "16") [ "LDFLAGS_EX_BE=-Wl,-export_dynamic" ]
```

Comment:
Should we make it explicit in the code, rather than a comment?  Just check if the major version is exactly 16?